### DISCUSSION TITLE
Managing duckdb Metadata with Gravitino

### DISCUSSION THREAD

**Discussion by xunliu** [duckdb/duckdb#10177] [Ideas]
Hi, folks

I'm [datastrato/gravitino](https://github.com/datastrato/gravitino/) developer.
Recent have a contributor created an issue https://github.com/datastrato/gravitino/issues/1358,
He hope use Gravitino to manage duckdb Metadata.
I think this is very interesting feature.

I know Duckdb is an embedded database,
First, I hope to know if Duckdb requires metadata management service ?
Also, If we integrated Duckdb in the Gravitino, I need via duckdb file path to get Metadata?

If you have any suggestions or opinions, Please tell me, I hope to hear from you. thanks!

**Comment by xunliu** [comment]
hi,
Has anybody given me some suggestions?

**Reply by adriens** [reply]
I'm also very intersted by metdatas, like column names, types ;-p

**Comment by Mause** [comment]
Can you clarify what metadata you're referring to here?

**Reply by qqqttt123** [reply]
Schema, table, view informations. Gravitino can manage multiple datasources, including relational datasource like Hive, MySQL, non-tabular datasource, like HDFS, streaming datasource, like Kafka.
